pkg/cmd/cluster/bind/bind.go

```diff
@@ -66,8 +67,8 @@ func NewBindCommand(f *factory.Factory) *cobra.Command {
 	cmd.Flags().BoolVarP(&opts.forceCreationWithoutAsk, "yes", "y", false, opts.localizer.MustLocalize("cluster.common.flag.yes.description"))
 	cmd.Flags().StringVarP(&opts.namespace, "namespace", "n", "", opts.localizer.MustLocalize("cluster.common.flag.namespace.description"))
 	cmd.Flags().BoolVar(&opts.ignoreContext, "ignore-context", false, opts.localizer.MustLocalize("cluster.common.flag.ignoreContext.description"))
-	cmd.Flags().BoolVar(&opts.forceOperator, "force-operator", false, opts.localizer.MustLocalize("cluster.bind.flag.forceOperator.description"))
-	cmd.Flags().BoolVar(&opts.forceSDK, "force-sdk", false, opts.localizer.MustLocalize("cluster.bind.flag.forceSDK.description"))
+	cmd.Flags().BoolVar(&opts.deploymentConfig, "deployment-config", false, opts.localizer.MustLocalize("cluster.bind.flag.useDeploymentConfig.description"))
```

Comment:
What about a flag which is generic and accepts multiple different types:

```shell
rhoas cluster connect --type=deployment|deploymentconfig
```

Reply:
Love it! Thank you for amazing suggestion. This is more future proof. 
deployment is default, DeploymentConfig is extra case that we were asked to support. 
We will not support both so app-type sounds like better flag

Reply:
Actually when implementing it I see that needs to be typed which requires much more than boolean enablement. 
Ideally we should be enabling both later but we cannot now due to bug in SBO. Leaving it as it is. Hope you do not mind.

Reply:
Don't mind no :) Merely a suggestion, but I don't have the domain knowledge to justify if it makes sense or not.